client/components/payment-method-logos/style.scss

```
@@ -0,0 +1,21 @@
+.connect-account-page__payment-methods--logos {
+	display: flex;
+	align-items: center;
+	border: 1px solid $gray-300;
+	border-bottom: none;
+	padding: 12px;
+	img {
+		margin-right: 8px;
+	}
+
+	&-count {
+		width: 38px;
+		height: 24px;
+		background-color: #f2f2f2;
```

In the paJDYF-dA0-p2 document, guidelines for background and text colors were provided. It would be beneficial to adhere to these recommendations. :)

```suggestion
		background-color: rgba($gray-700, 0.10);
		color: $gray-900;
```

---

changed in [a8827d3](https://github.com/Automattic/woocommerce-payments/pull/8977/commits/a8827d358ba715c9bb02527de2e03d93e75ad6f7) . thx.